api/lib/Converter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Ockham.Data
+{
+    public delegate object ConverterDelegate(object value, Type targetType, ConvertOptions options);
```

Comment:
We can probably drop `targetType` since it is unlikely that it would be needed.

Reply:
That's a good point. I had been thinking of complex cases where `targetType` might be some derived type...but that implies mechanics we probably don't even want to implement.